tools/scope_upload_classification.py

```diff
@@ -307,4 +355,5 @@ def upload_classification(
         args.ztf_origin,
         args.skip_phot,
         args.p_threshold,
+        not args.no_match_ids,
```

Comment:
The goal here was to default to the new ID-matching behavior with no argument necessary, but to have an easy way to perform the old search method using a single argument. Ideally this would be a `-match_ids` argument defaulting to True and able to be set False by the user. I had already written the `match_ids` keyword into the function that's called, but passing in False to argparse still returned True (presumably because it does not convert the string 'False' to boolean). That's why I employed this workaround, but it's probably more straightforward just to change the keyword in the `upload_classification` function.

Reply:
Whatever you think is better.

Reply:
I'll go with using the `no-match-ids` keyword across the board. Most uploads should use the new behavior, so I think it's best to associate the argument for the exceptional cases.